### ISSUE TITLE
Show multiple spinners at the same time

### ISSUE THREAD

**Issue by hangyas** [charmbracelet/gum#370]
I'm working on a script that runs multiple sub-tasks in parallel, it would be nice if I could wrap them with `gum spin` so all of the subtasks would have their own spinners displayed at the same time (similar to when docker is pulling image layers in parallel)

Maybe it would be something to be achieved with `gum join`, but as far as I can tell it's not possible at the moment

**Comment by caarlos0** [comment]
I think it would be very hard to do this in a shell script. At this point, probably easier to make an actual bubbletea application, maybe using [huh](https://github.com/charmbracelet/huh).

That said, this is out of scope for us, closing.

Thanks!
